packages/web-app/src/modules/auth/AuthStore.tsx

```diff
@@ -61,23 +21,9 @@ export class AuthStore {
     try {
       await this.axios.get('/api/v1/profile').then(() => runInAction(() => (this.isAuthenticated = true)))
     } catch {
-      if (this.loginPromise) {
-        return this.loginPromise
-      }
+      const routeWithoutTrailingSlash = this.router.location.pathname.slice(1)
 
-      // Create a promise that we will return
-      this.loginPromise = new Promise(async (resolve, reject) => {
-        // Save the promise data to reject
-        this.loginResolve = resolve
-        this.loginReject = reject
-
-        // Save current route so we can go back to it later once it is resolved
-        this.startingRoute = this.router.location.pathname === '/login' ? '/store' : this.router.location.pathname
-
-        this.router.replace('/login')
-      })
-
-      await this.loginPromise
+      window.location.replace(`${config.loginUrl}?redirect_webapp=${routeWithoutTrailingSlash}`)
```

Comment:
Why did you use window.location.replace? 

```suggestion
      this.router.replace(`/login?redirect_webapp=${routeWithoutTrailingSlash}`)
```

Reply:
it will not work locally

Reply:
Even if I run Login app locally?

Reply:
you can't run 2 apps on the same port

Reply:
but also for the development it doesn't make sense to always start login app along with web

Reply:
@Maks19 changed